client/resource_group/controller/controller.go

```diff
@@ -300,6 +300,15 @@ func (c *ResourceGroupsController) Start(ctx context.Context) {
 					watchMetaChannel, err = c.provider.Watch(ctx, pd.GroupSettingsPathPrefixBytes, opt.WithRev(metaRevision), opt.WithPrefix(), opt.WithPrevKV())
 					if err != nil {
 						log.Warn("watch resource group meta failed", zap.Error(err))
+						// Stop the timer if it's not stopped.
+						if !watchRetryTimer.Stop() {
```

Comment:
Does master need it?

Reply:
I think we can merge this pr to make it easier to pick to other branches, and then delete all the stop+drain+reset usage with a new pr.

Reply:
I prefer to directly fix on the corresponding branch.